storage/docker-compose.yml

```diff
@@ -33,7 +34,7 @@ services:
             ./rpc.py bdev_malloc_create -b Malloc1 64 512 && \
             ./rpc.py nvmf_create_transport -t TCP -u 8192 -m 4 -c 0  && \
             ./rpc.py nvmf_create_subsystem nqn.2016-06.io.spdk:cnode1 -a -s SPDK00000000000001 -d SPDK_Controller1  && \
-            ./rpc.py nvmf_subsystem_add_listener nqn.2016-06.io.spdk:cnode1 -t tcp -a 127.0.0.1 -f ipv4 -s 4444  && \
+            ./rpc.py nvmf_subsystem_add_listener nqn.2016-06.io.spdk:cnode1 -t tcp -a  `hostname -i` -f ipv4 -s 4444  && \
```

Comment:
Wouldn't this be better to just put the docker hostname here? Docker will resolve local DNS entries for you.

Reply:
I wanted to, but I can't... SPDK doesn't accept hostnames, only valid IP addresses 
https://spdk.io/doc/jsonrpc.html#nvmf_subsystem_add_listener

Reply:
Well, you could resolve the IP.  Something like -a `host [the hostname]`

Reply:
this is what I did above, no ? `-a  `hostname -i` `